Mapping/inc/Obstacles.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Header file providing class for obstacles
+ * Author: Frederick Wachter
+ */
+
+#ifndef __OBSTACLES__
+#define __OBSTACLES__
+
+#include <array>
+using namespace std;
+
+class Obstacles {
+	public:
+		bool *directions[4] = { &north, &east, &south, &west };
```

Comment:
Actually it may be better style to do all initializations in the constructor and not on the member declarations themselves.